meson.build

```diff
@@ -0,0 +1,200 @@
+project('r2inference', ['c','cpp'],
+  version : '0.6.0',
+  meson_version : '>= 0.50',)
+
+project_name = meson.project_name()
+project_version = meson.project_version()
+version_arr = project_version.split('.')
+version_major = version_arr[0].to_int()
+version_minor = version_arr[1].to_int()
+version_micro = version_arr[2].to_int()
+if version_arr.length() == 4
+  version_nano = version_arr[3].to_int()
+else
+  version_nano = 0
+endif
+version_is_dev = version_minor % 2 == 1 and version_micro < 90
+
+# Get an object returns describing a compiler
+cc = meson.get_compiler('c')
+
+# Find external dependencies
+if get_option('enable-tensorflow')
+  tensorflow = cc.find_library('tensorflow', required: true)
+  tensorflow_dep = declare_dependency(dependencies: tensorflow, include_directories: '/usr/local/include/tensorflow/')
+endif
+
+if get_option('enable-tflite')
+  tensorflow_lite = cc.find_library('tensorflow-lite', required: true)
+  tensorflow_lite_dep = declare_dependency(dependencies: tensorflow_lite)
```

Comment:
Same case here with the dependency function.

Reply:
Same as reply above.